src/main/msp/msp.c

```diff
@@ -1793,6 +1793,15 @@ static bool mspProcessOutCommand(int16_t cmdMSP, sbuf_t *dst)
 #else
         sbufWriteU8(dst, 0);
 #endif
+        // Added in MSP API 1.43
```

Comment:
This will be in 1.44, to go with Betaflight 4.3.

Reply:
@mikeller - Could we maybe include these in 1.43 to go with 4.2?

Reply:
@ctzsnooze: We've been bitten time and again by adding new parameters to MSP early, and then needing to add complex code to configurator to work around later changes in parameter format - this is why we use a one-release-cycle stand down period for this now.

Reply:
Feed forward averaging and boost are in Betaflight 4.1 already, so this is one release after addition. Including it now, would be consistent with the timing of RPM filter or GPS rescue.

Reply:
@bizmar: Can you provide the changesets to prove the claims that you are making?

Reply:
Sure, I'm not familiar with github enough to know what a changeset is, I hope links to PRs from release notes are good enough.

Feedforward 2.0 (4.1) https://github.com/betaflight/betaflight/pull/8623

RPM filter (4.0) https://github.com/betaflight/betaflight/pull/7271
RPM filter MSP (4.1) https://github.com/betaflight/betaflight/pull/8817

GPS rescue (3.4) https://github.com/betaflight/betaflight/pull/5753 (this one has many PRs, but this one is linked in the release notes)
GPS rescue MSP (3.5) https://github.com/betaflight/betaflight/pull/5955

Reply:
@bizmar: Why are you making claims if you don't even know how to verify that they are actually correct. I am certainly not going to do the work for you to try and prove your unsubstantiated claims. Please stop wasting people's time in this project.

Reply:
I'm sorry I'm not able to satisfy what you asked for. Could you please clarify what you want. I don't see how diffs between versions would help (that's my interpretation of changeset, google was not helpful) 

I linked to the PRs linked in the release notes of each BF version, they have Assigned milestones and merge dates. And to the PRs that implemented MSP for those features, that also have millstones and merged dates associated.

Reply:
@bizmar: I want you to bring proof for the unsubstantiated claims, or else stop making unsubstantiated claims that do nothing but waste everybody's time.